src/view/search-results.inc

```diff
@@ -0,0 +1,128 @@
+    {%-  for langstring, labels in foreignLabels  %}
+      {%- for label in labels.prefLabel|default([])|merge(labels.altLabel|default([])) %}
+    <span class="versal{% if label.type == 'skos:prefLabel' %} prefLabel{% elseif label.type == 'skos:altLabel' %} replaced{% endif %} proplabel">{{ label }}</span>{%- if label.lang %}<span class="versal"> ({{ label.lang }})</span>{% endif -%}{%- if loop.last == false %}<span class="versal">, </span>{% endif -%}
+      {% endfor -%}
+      {%- if loop.last == false %}<span class="versal">, </span>{% endif -%}
+    {% endfor -%}
+    </li>
+{% endif -%}
+{%- if concept.vocab.config.additionalSearchProperties %}
+  {% set mappingProperties = concept.mappingProperties(concept.vocab.config.additionalSearchProperties) %}
+  {%~ if mappingProperties ~%}
+    {%~ for property in mappingProperties ~%} {# loop through ConceptProperty objects #}
+      {%~ if property.type in concept.vocab.config.additionalSearchProperties ~%}
+      <li class="list-group-item px-0 py-1">
+        <span class="property-click skosmos-tooltip-wrapper skosmos-tooltip t-top" data-title="{{ property.type }}">
+          <i class="property-hover fa-solid fa-globe"></i>
+        </span>
+        <div class="property-values">
+          {%~ for propval in property.values ~%} {# loop through ConceptMappingPropertyValue objects #}
+          {%~ if propval.exVocab ~%}<span class="redirected-vocab-id prefLabel">{{ propval.exVocab.shortName }}: </span>{%~ endif ~%}<span class="versal value">{{ propval.label(request.contentLang) }}</span>{%~ if propval.label(request.contentLang).lang != request.contentLang ~%}<span class="versal value"> ({{ propval.label(request.contentLang).lang }})</span>{%~ endif ~%}{%~ if loop.last == false ~%}<span class="versal">, </span>{%~ endif ~%}
+          {%~ endfor ~%}
+        </div>
+      </li>
+      {%~ endif ~%}
+    {%~ endfor ~%}
+  {%~ endif ~%}
+{% endif -%}
+{%- for property in conceptProperties %} {# loop through ConceptProperty objects #}
+  {%~ if property.type == 'rdf:type' ~%}
+    <li class="list-group-item px-0 py-1">
+      <span class="property-click skosmos-tooltip-wrapper skosmos-tooltip t-top" data-title="concept_types">
+        <i class="property-hover fa-solid fa-arrows-to-circle"></i>
+      </span>
+      <span class="property-values">
+      {%~ for propval in property.values ~%} {# loop through ConceptPropertyValue objects #}
+        <span class="versal value">{{ propval.label }}</span>
+      {%~ endfor ~%}
+      </span>
+    </li>
+  {%~ endif ~%}
+{% endfor -%}
+{%~ if concept.uri ~%}
+    <li class="list-group-item px-0 py-1">
+      <span class="URI-icon">URI</span>
+      <span class="search-result-uri">{{ concept.uri }}</span>
+      <i class="fa-solid fa-copy"></i> <!-- no copy to clipboard functionality yet -->
```

Comment:
Please open a new issue about the missing copy functionality

Reply:
Opened, #1504 .